lib/create/attach-to-issue-from-message.js

```diff
@@ -0,0 +1,219 @@
+        }],
+      });
+    }
+
+    const permissions = {
+      issues: 'write',
+      pull_requests: 'write',
+    };
+    const { owner, repo } = resource;
+    if (!(await Installation.assertPermissions(await robot.auth(), { owner, repo, permissions }))) {
+      const appUrl = (await robot.info()).html_url;
+      return res.send({
+        errors: [{
+          name: 'url',
+          error: `Insufficient permissions. Please make sure the Slack app is installed on this repository: ${appUrl}`,
+        }],
+      });
+    }
+  }
+
+  let data;
+  try {
+    if (url) {
+      ({ data } = await slackUser.GitHubUser.client.issues.createComment({
+        ...resource,
+        type: undefined,
+        body: comment,
+      }));
+    } else {
+      const addComment = `
+        mutation comment($id: ID!, $body: String!) {
+          addComment(input: {subjectId: $id, body: $body}) {
+            commentEdge {
+              node {
+                url
+              }
+            }
+          }
+        }`;
+      data = (await slackUser.GitHubUser.client.query(
+        addComment,
+        { id: issueId, body: comment },
+      ));
+    }
+  } catch (err) {
+    if (!(err.name && err.name === 'GraphQLError')) {
+      throw err;
+    }
+    req.log.debug({ err }, 'Could not create comment');
+
+    const appUrl = (await robot.info()).html_url;
+    return res.send({
+      errors: [{
+        name: 'issueId',
+        error: `Please make sure the Slack app has been granted issue and pull request write access on this repository: ${appUrl}`,
+      }],
+    });
+  }
+
+  let issueUrl;
+  if (data.html_url) {
+    issueUrl = data.html_url;
+  } else {
+    issueUrl = data.addComment.commentEdge.node.url;
+  }
+
+  await axios.post(response_url, {
+    response_type: 'ephemeral',
+    attachments: [new CommentCreated(issueUrl).getAttachment()],
+  });
+  return res.send();
+}
+
+async function loadIssuesAndPrs(req, res) {
+  function formatLabel(issue, includeRepository = false) {
+    const label = `#${issue.number} ${issue.title}`;
+    if (includeRepository) {
+      return `[${issue.repository.nameWithOwner}] ${label}`;
+    }
+    if (label.length > 30) {
+      return `${label.slice(0, 30).slice(0, -3)}...`;
+    }
+    return label;
+  }
+
+  const { slackUser } = res.locals;
+  const { value } = req.body;
+
+  const query = `
+    query {
+      viewer {
+        issues(first: 100, orderBy: {field: UPDATED_AT, direction: DESC}) {
+          nodes {
+            title
+            number
+            id
+            repository {
+              nameWithOwner
+            }
+          }
+        }
+      pullRequests(first: 100, orderBy: {field: UPDATED_AT, direction: DESC}) {
+        nodes {
+          title
+          number
+          id
+          repository {
+            nameWithOwner
+          }
+        }
+      }
+    }
+  }`;
+
+  const { data } = await slackUser.GitHubUser.client.request({
```

Comment:
Yep, I'll add a comment. For posterity, it's because of the reasons described here: https://github.com/probot/probot/issues/484#issuecomment-393003393